docs/en/getting-started/docker/docker-compose.yml

```diff
@@ -1,13 +1,13 @@
-version: '2.2'
+version: "2.2"
 services:
-  es01:
+  es-node01:
     image: docker.elastic.co/elasticsearch/elasticsearch:{version}
-    container_name: es01
+    container_name: es-node01
     environment:
```

Comment:
I suggest that if we're keeping this compose file around, we should make it obvious that TLS is not enabled, and do so by setting xpack.security.http.ssl.enabled=false and xpack.security.transport.ssl.enabled=false here. Autoconfiguration won't be triggered because of the other settings, but I suggest we make this explicit too by setting xpack.security.autoconfiguration.enabled=false

Reply:
I didn't read the PR description so this comment is most probably relevant for the follow up PR.